code/clue.js

```diff
@@ -1,59 +1,274 @@
+  age: 55,
+  image: "./assets/plum.png",
+  occupation: "Professor",
+  favouriteWeapon: "knife",
+};
 
+const missScarlet = {
+  firstName: "Cassandra",
+  lastName: "Scarlett",
+  color: "#e97171",
+  description: "She likes to create nail paintings",
+  age: 48,
+  image: "./assets/scarlet.png",
+  occupation: "Nail painter",
+  favouriteWeapon: "candlestick",
+};
+
+const mrsPeacock = {
+  firstName: "Elenor",
+  lastName: "Peacock",
+  color: "blue",
+  description: "She likes to teach kids history",
+  age: 58,
+  image: "./assets/peacock.png",
+  occupation: "Teacher",
+  favouriteWeapon: "dumbbell",
+};
+
+const colonelMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  color: "yellow",
+  description: "He has just one leg",
+  age: 74,
+  image: "./assets/mustard.png",
+  occupation: "Plummer",
+  favouriteWeapon: "poison",
+};
+
+const mrsWhite = {
+  firstName: "Mrs",
+  lastName: "White",
+  color: "white",
+  description: "She do as little as possible",
+  age: 82,
+  image: "./assets/white.png",
+  occupation: "Relaxing",
+  favouriteWeapon: "axe",
+};
+
+// Weapon objects
 const rope = {
-  name: 'rope',
-  weight: 10
-}
+  name: "rope",
+  id: "rope",
+  weight: 10,
+};
+
+const knife = {
+  name: "knife",
+  id: "knife",
+  weight: 2,
+};
+
+const candleStick = {
+  name: "candlestick",
+  id: "candlestick",
+  weight: 6,
+};
+
+const dumbBell = {
+  name: "dumbbell",
+  id: "dumbbell",
+  weight: 4,
+};
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const poison = {
+  name: "poison",
+  id: "poison",
+  weight: 1,
+};
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const axe = {
+  name: "axe",
+  id: "axe",
+  weight: 5,
+};
 
+const bat = {
+  name: "bat",
+  id: "axe",
+  weight: 3,
+};
+
+const throphy = {
+  name: "throphy",
+  id: "throphy",
+  weight: 8,
+};
+
+const pistol = {
+  name: "pistol",
+  id: "pistol",
+  weight: 7,
+};
+
+// Arrays for suspects, weapons and rooms
 const suspects = [
   mrGreen,
-  mrsWhite
-  // ...  and the rest
-]
+  professorPlum,
+  missScarlet,
+  mrsPeacock,
+  colonelMustard,
+  mrsWhite,
+];
+
+const weapons = [
+  rope,
+  knife,
+  candleStick,
+  dumbBell,
+  poison,
+  bat,
+  throphy,
+  pistol,
+];
 
-const weapons = []
+const rooms = [
+  "Dinning Room",
+  "Conservatory",
+  "Kitchen",
+  "Study",
+  "Library",
+  "Billiard Room",
+  "Lounge",
+  "Ballroom",
+  "Hall",
+  "Spa",
+  "Living Room",
+  "Observatory",
+  "Theater",
+  "Guest House",
+  "Patio",
+];
 
-const rooms = []
+const randomSelector = (array) => {
+  return array[Math.floor(Math.random() * array.length)];
+};
 
-// THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
-const randomSelector = array => {
-  return array[Math.floor(Math.random() * array.length)]
-}
+// Mystery object
+const mystery = {
+  killer: null,
+  weapon: null,
```

Comment:
Great job again, it's the best option to initialise object with `null` values if you want to overwrite them later on :)